tests/unit/conftest.py

```diff
@@ -46,13 +47,16 @@ def pyramid_settings():
 
 
 @pytest.fixture
-def make_pyramid_request(pyramid_config):
-    def _make_pyramid_request(path):
+def make_request(pyramid_config):
+    def make_request(path="/irrelevant", params=None):
+        if params:
+            path += "?" + urlencode(params)
```

Comment:
Everywhere we use this we end up doing params based stuff. So I've added it into the base fixture and given it a snappier name.